documentation/modules/metrics/kafka-bridge/proc_kafka-bridge-enabling-grafana.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// metrics/assembly_metrics-kafka-bridge.adoc
+
+[id='proc-kafka-bridge-enabling-{context}']
+
+= Enabling the Kafka Bridge Grafana dashboard
+
+If you deployed Kafka Bridge with your Kafka cluster, you can enable Grafana to present the metrics data it exposes.
+
+A Kafka Bridge dashboard is provided in the `examples/metrics` directory as a JSON file:
+
+* `strimzi-kafka-bridge.json`
+
+.Prerequisites
+
+* Kafka Bridge is deployed with xref:proc-kafka-bridge-configuring-{context}[Kafka Bridge metrics configuration]
+* xref:assembly-metrics-prometheus-{context}[Prometheus and Prometheus Alertmanager are deployed to the Kafka cluster]
+* xref:proc-metrics-deploying-grafana-{context}[Grafana is deployed to the Kafka cluster]
+
+This procedure assumes you already have access to the Grafana user interface and Prometheus has been added as a data source.
+If you are accessing the user interface for the first time, see xref:assembly-metrics-grafana-{context}[Grafana].
+
+.Procedure
+
+. Access the Grafana user interface.
```

Comment:
Step as link to https://strimzi.io/docs/operators/master/deploying.html#proc-metrics-grafana-dashboard-str

Reply:
What do you mean?

Reply:
I don't see it as a link in the Kafka Exporter section for example.